build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -737,8 +737,8 @@ message ActionResult {
   // The output files of the action. For each output file requested in the
   // `output_files` field of the Action, if the corresponding file existed after
   // the action completed, a single entry will be present either in this field,
-  // or in the output_file_symlinks field, if the file was a symbolic link to
-  // another file.
+  // or the `inlined_output_files` field or the `output_file_symlinks` field if the
```

Comment:
Out of date? inlined_output_files no longer exists, I don't think.

Reply:
done.